.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v1
       with:
-        node-version: 14.x
+        node-version: 20.x
```

Comment:
modern versions of rollup do not support node < 18. Since [Node.js v20](https://nodejs.org/en/blog/release/v20.9.0) is available and LTS, we are upgrading the action runner to use Node v20